injitest/src/main/java/inji/pages/AddNewCardPage.java

```diff
@@ -29,6 +29,9 @@ public class AddNewCardPage extends BasePage{
     @AndroidFindBy(uiAutomator = "new UiSelector().textContains(\"Mangyaring piliin ang iyong gustong tagabigay mula sa mga opsyon sa ibaba upang magdagdag ng bagong card.\")")
     private WebElement addNewCardGuideMessageInFillpino;
     
+    @AndroidFindBy(uiAutomator = "new UiSelector().textContains(\"नया कार्ड जोड़ने के लिए कृपया नीचे दिए गए विकल्पों में से अपना पसंदीदा जारीकर्ता चुनें।\")")
+    private WebElement addNewCardGuideMessageInHindi;
```

Comment:
Dummy Logic:
Page Class:
verifyLanguageForGuideMessage(String language){
String actualText = locator.getText();


Boolean method(String expectedLanguage){

switch(language)

Case Hindi:
return boolean (expected text in Hindi language, actualText)

Case English:

}

}

@Test{
method(expectedLanguage)
}

Reply:
added